Dockerfile.node

```diff
@@ -27,14 +27,15 @@ FROM eu.gcr.io/stashify-218417/basenode:latest as builder
 
 ARG CARGO_FEATURES=release-feature-set
 ARG CARGO_PROFILE=release
+ENV RUSTFLAGS="--cfg prometheus_metrics"
```

Comment:
You should propagate this changes to Dockerfile.local_node, the version of Dockerfile.node that don't recall on our private basenode